src/main/java/net/openhft/chronicle/testframework/internal/StandardExceptionTracker.java

```diff
@@ -0,0 +1,117 @@
+package net.openhft.chronicle.testframework.internal;
+
+import org.jetbrains.annotations.NotNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+/**
+ * A test utility class for recording and executing assertions about the presence (or absence) of exceptions
+ *
+ * @param <T> The class used to represent thrown exceptions
+ */
+public final class StandardExceptionTracker<T> implements ExceptionTracker<T> {
```

Comment:
I wonder if it should be called "Vanilla" ?

Reply:
Yes, I think that is more Chronicle style.